packages/driver/src/cypress/stack_utils.ts

```diff
@@ -482,16 +482,19 @@ const normalizedUserInvocationStack = (userInvocationStack) => {
   // add/$Chainer.prototype[key] (cypress:///../driver/src/cypress/chainer.js:30:128)
   // whereas Chromium browsers have the user's line first
   const stackLines = getStackLines(userInvocationStack)
-  const winnowedStackLines = _.reject(stackLines, (line) => {
-    // WARNING: STACK TRACE WILL BE DIFFERENT IN DEVELOPMENT vs PRODUCTOIN
+  const windowedStackLines = _.reject(stackLines, (line) => {
```

Comment:
```suggestion
  const nonCypressStackLines = _.reject(stackLines, (line) => {
```

"winnowed", as in "removed", was originally intended but doesn't seem appropriate now that I think about it, since we're getting the lines we want and not the removed lines.

Reply:
Oh thanks! figured it was a typo!

Reply:
[`4fe9cf0` (#24511)](https://github.com/cypress-io/cypress/pull/24511/commits/4fe9cf0ae1644bea7ea2fc00fa06d3986e63195f)